releases/release-0.4/README.md

```diff
@@ -0,0 +1,11 @@
+# Konveyor v0.4
+
+## Summary (Tentative)
+
+The 0.4 release cycle is as follows:
+
+- **Tuesday 2 January 2024**: Release begins
```

Comment:
should we move this by a week and start on jan 9th? Allow time for folks coming back from holidays and catch up?